.coveragerc

```diff
@@ -461,8 +461,6 @@ omit =
     homeassistant/components/freebox/camera.py
     homeassistant/components/freebox/home_base.py
     homeassistant/components/freebox/switch.py
-    homeassistant/components/fritz/common.py
-    homeassistant/components/fritz/device_tracker.py
```

Comment:
homeassistant/components/fritz/coordinator.py is on 75%
homeassistant/components/fritz/device_tracker.py is on 98%